fgrpc/grpcrunner.go

```diff
@@ -35,9 +35,11 @@ import (
 
 const (
 	// DefaultGRPCPort is the Fortio gRPC server default port number.
-	DefaultGRPCPort = "8079"
-	prefixHTTP      = "http://"
-	prefixHTTPS     = "https://"
+	DefaultGRPCPort  = "8079"
+	defaultHTTPPort  = "80"
+	defaultHTTPSPort = "443"
+	prefixHTTP       = "http://"
+	prefixHTTPS      = "https://"
```

Comment:
those must exist somewhere but I think it's fine to clone here if we haven't easily found those constants

Reply:
`prefixHTTP` and `prefixHTTP` were already part of the code. They show up in the diff b/c `=` is pushed over a couple spaces. I was thinking the same thing. I looked through the net, net/http and net/url pkgs and did not see any constants for the standard http/https ports.

Reply:
I meant somewhere in some go standard library